x-pack/plugins/infra/public/utils/url_state.tsx

```diff
@@ -155,16 +155,15 @@ export const replaceStateKeyInQueryString = <UrlState extends any>(
   stateKey: string,
   urlState: UrlState | undefined
 ) => (queryString: string) => {
-  const previousQueryValues = parse(queryString, { sort: false });
+  const previousQueryValues = parse(queryString);
   const encodedUrlState =
     typeof urlState !== 'undefined' ? encodeRisonUrlState(urlState) : undefined;
 
   return stringify(
```

Comment:
I believe we can again use `makeUrlFromQuery` here to keep the old (preferable) behaviour. This also stops the changes to the `link_to/**.test.tsx` test files being needed.